test/fixtures/test-runner/test/random.cjs

```diff
@@ -2,3 +2,6 @@
 const test = require('node:test');
 
 test('this should pass');
+
+test('this should pass too');
```

Comment:
I would be surprised if this change didn't break other tests, but I guess the CI will let us know for sure.